### ISSUE TITLE
Removes too many parentheses in macro definition

### ISSUE THREAD

**Issue by rlipscombe** [AdRoll/rebar3_format#293]
From https://github.com/erlang-ls/erlang_ls/issues/1168

I've got the following macro:

```erlang
% note the parens:
-define(METRICS, (application:get_env(?APP, metrics_module, ?DEFAULT_METRICS_MODULE))).
%                ^                                                                  ^
```

...which I use like this:

```erlang
notify_cache_hit() -> ?METRICS:increment(cache_hits).
```

When I run the formatter, it removes the parentheses, giving me this:

```erlang
% this is wrong:
-define(METRICS, application:get_env(?APP, metrics_module, ?DEFAULT_METRICS_MODULE)).
```

This is incorrect, and causes `syntax error before: ':'` errors when I attempt to use the macro.

I've confirmed that this happens with rebar3_format v1.0.1, outside of ErlangLS.

**Comment by elbrujohalcon** [comment]
Thanks for the report, @rlipscombe … It's likely related to #74 and #216.

Macros with code in them are generally a pain point for the formatter.

**Comment by elbrujohalcon** [comment]
Yes. This project will soon be archived, anyway, because of #366. I recommend you to move to [erlfmt](https://github.com/WhatsApp/erlfmt)
